src/js/angular-tooltips.js

```diff
@@ -77,12 +79,7 @@
 
         thisElement.bind(showTriggers, function onMouseEnterAndMouseOver() {
 
-          if (!initialized) {
-
-            initialized = true;
-            $scope.initTooltip(side);
-          }
-
+          $scope.initTooltip(side);
```

Comment:
But I think checking element movements costs more than initializing tooltip before showing.  If you have better solution, let me know please.